app/src/renderer/components/common/TmPage.vue

```diff
@@ -9,7 +9,7 @@
       </h3>
       <slot slot="menu-body" name="menu-body">
         <tm-balance v-if="session.signedIn" />
-        <tool-bar :refresh="refreshable" />
+        <tool-bar />
```

Comment:
I think if we want to kill this feature you should also remove from all the pages where it's used + the function definitions.
Do we want to remove the refresh button?

Reply:
Now that I think about it this should be a separate PR 🚀

Reply:
Sorry I can't resist
![oops](https://user-images.githubusercontent.com/1656471/53554839-f8d27a80-3b40-11e9-87d5-0518bbae49fb.gif)

Reply:
👎 please put the refresh back. as long as we don't trust the UI is updating everything nicely we should have this.

Reply:
> as long as we don't trust the UI is updating everything nicely we should have this.

if a UI team can't trust their UI — WHAT ARE WE EVEN DOING HERE

---

sorry, i will add the refresh back

Reply:
we are scientists not believers ;)